app/WorkoutDay.tsx

```diff
@@ -5,10 +5,14 @@ import { useRoute, useNavigation, RouteProp } from '@react-navigation/native';
 import { ThemedView } from '@/components/ThemedView';
 import { ThemedText } from '@/components/ThemedText';
 import { RootStackParamList } from '@/types/navigation'; // Import your navigation types
+import { useRouter } from 'expo-router';
 
 export default function WorkoutDayScreen() {
   const route = useRoute<RouteProp<RootStackParamList, 'WorkoutDay'>>(); // Use the route types
-  const navigation = useNavigation();
+
+  // const navigation = useNavigation();
+  const router = useRouter();
```

Comment:
You can import 'router' and use the router.push method that way instead of having to define the hook. It's a little simpler that way